src/ngraph_builder.cc

```diff
@@ -1081,14 +1126,16 @@ static Status TranslateConv2DOp(
       ng_input, ng_filter, ng_strides, ng_dilations, ng_padding_below,
       ng_padding_above);
 
+  SaveNgOp(first_ng_op_map, op->name(), ng_conv);
```

Comment:
Did we not save the entry nodes already here: "SaveNgOp(first_ng_op_map, op->name(), ng_filter);" ?

Reply:
will check

Reply:
Yes, corrected.